metatensor-torch/src/atomistic/model.cpp

```diff
@@ -1032,6 +1034,29 @@ static std::unordered_map<std::string, Quantity> KNOWN_QUANTITIES = {
         {"J", "Joule"},
         {"Ry", "Rydberg"},
     }}},
+    {"energy_ensemble", Quantity{/* name */ "energy", /* baseline */ "eV", {
```

Comment:
This should not be needed, `energy_ensemble` should be able to re-use the `energy` quantity and associated units